registry.ts

```diff
@@ -239,6 +298,40 @@ export class Denopkg implements RegistryUrl {
   regexp = /https?:\/\/denopkg.com\/[^\/\"\']*?\/[^\/\"\']*?\@[^\'\"]*/;
 }
 
+export class PikaScope implements RegistryUrl {
+  url: string;
+
+  parts(): PackageInfo {
+    return defaultParts(this);
+  }
+
+  constructor(url: string) {
+    this.url = url;
+  }
+
+  async all(): Promise<string[]> {
+    const { scope, packageName } = this.parts();
+    return await unpkgVersions(`${scope}/${packageName}`);
+  }
+
+  at(version: string): RegistryUrl {
+    const { parts, packageName } = this.parts();
+    parts[4] = `${packageName}@${version}`;
+    return new PikaScope(parts.join("/"));
+  }
+
+  version(): string {
+    const { version } = this.parts();
+    if (version === undefined) {
+      throw Error(`Unable to find version in ${this.url}`);
```

Comment:
won't this be thrown already in defaultParts? https://github.com/hayd/deno-udd/pull/53/files#diff-58f42611a053bb45add25a11c10ed01c89fa3d17a917ecea2de4fad160568ff8R173

Reply:
Oh, that's right. I should have paid more attention.
Thank you for your pointing. I'll fix it.